### PR TITLE
add field capabilities

### PR DESCRIPTION
**Pull request by MarinaRazumovsky**

### CODE REVIEW

File: jest-common/src/test/java/io/searchbox/fields/FieldsStatsTest.java [hunk 1]
```diff
@@ -16,25 +16,25 @@
 
     @Test
     public void testBasicUriGeneration() {
-        FieldStats fieldStats = new FieldStats.Builder(FIELDS).setIndex(INDEX).build();
```
Comment (ofir76) [inline jest-common/src/test/java/io/searchbox/fields/FieldsStatsTest.java]: why we remove field stats tests?

File: jest/src/test/java/io/searchbox/fields/FieldCapabilitiesIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.SUITE, numDataNodes = 1)
+public class FieldCapabilitiesIntegrationTest extends AbstractIntegrationTest {
+
+    private static final String INDEX = "twitter";
+    private static final String TYPE = "tweet";
+    private static final String TEST_FIELD = "test_name";
+    private static final List FIELDS = Collections.singletonList(TEST_FIELD);
+
+    @Test
+    public void testFieldStats() throws IOException {
```
Comment (ofir76) [inline jest/src/test/java/io/searchbox/fields/FieldCapabilitiesIntegrationTest.java]: testFieldCaps

**Review by ofir76 — CHANGES_REQUESTED** [review]
👍 ,2 comments

File: jest-common/src/test/java/io/searchbox/fields/FieldsStatsTest.java (hunk 1 above)
Comment (MarinaRazumovsky) [reply jest-common/src/test/java/io/searchbox/fields/FieldsStatsTest.java]: It looks like merge conflict from 6.3. I reverted FieldsStatsTest and added FieldCapabilitiesTest

**Review by ofir76 — APPROVED** [review]

**Comment by asafm** [comment]
@MarinaRazumovsky Please don't forget to submit a PR to Jest so can keep the using original Jest and not our own customised version

**Pull request merged**